run-tests.js

```diff
@@ -266,54 +266,55 @@ const testExchange = async (exchange) => {
         args.push ('--info')
     }
     const allTestsWithoutTs = [
-            { language: 'JavaScript',     key: '--js',           exec: ['node',      'js/src/test/test.js',           ...args] },
-            { language: 'Python 3',       key: '--python',       exec: ['python3',   'python/ccxt/test/test_sync.py',  ...args] },
-            { language: 'Python 3 Async', key: '--python-async', exec: ['python3',   'python/ccxt/test/test_async.py', ...args] },
-            { language: 'PHP',            key: '--php',          exec: ['php', '-f', 'php/test/test_sync.php',         ...args] },
-            { language: 'PHP Async',      key: '--php-async',    exec: ['php', '-f', 'php/test/test_async.php',   ...args] },
-        ]
-
-        if (!skipSettings[exchange] || !skipSettings[exchange].skipCSharp) {
-            allTestsWithoutTs.push (
-                { language: 'C#',             key: '--csharp',        exec: ['dotnet', 'run', '--project', 'cs/tests/tests.csproj',               ...args] },
-            )
-        }
-        // if it's not WS tests, then add sync versions to tests queue
-        if (!wsFlag) {
-            allTestsWithoutTs.push (
-                { language: 'PHP',            key: '--php',          exec: ['php', '-f', 'php/test/test_sync.php',         ...args] } 
-            );
-            allTestsWithoutTs.push (
-                { language: 'Python 3',       key: '--python',       exec: ['python3',   'python/ccxt/test/test_sync.py',    ...args] }
-            );
-        }
+        { language: 'JavaScript',     key: '--js',           exec: ['node',      'js/src/test/test.js',            ...args] },
+        { language: 'Python 3 Async', key: '--python-async', exec: ['python3',   'python/ccxt/test/test_async.py', ...args] },
+        { language: 'PHP Async',      key: '--php-async',    exec: ['php', '-f', 'php/test/test_async.php',        ...args] },
+    ]
+
+    // if it's not WS tests, then add sync versions to tests queue
+    if (!wsFlag) {
+        allTestsWithoutTs.push (
+            { language: 'PHP',        key: '--php',          exec: ['php', '-f', 'php/test/test_sync.php',         ...args] } 
+        );
+        allTestsWithoutTs.push (
+            { language: 'Python 3',   key: '--python',       exec: ['python3',   'python/ccxt/test/test_sync.py',  ...args] }
+        );
+    }
 
-        const allTests = allTestsWithoutTs.concat([
```

Comment:
i have not removed the codes from here down below, just git shows them into differences

Reply:
so, there are no changed from here down below, other than one indent was removed (which was extra)